src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/UnstructuredAddress.java

```diff
@@ -35,9 +35,9 @@ public static UnstructuredAddress fromString(final String edifact) {
         final String[] splitByColon = Split.byColon(splitByPlus[INDEX_ADDRESS]);
 
         String[] addressLines = null;
-        if (splitByColon.length > ADDRESS_LINES) {
-            addressLines = new String[ADDRESS_LINES];
-            System.arraycopy(splitByColon, 1, addressLines, 0, ADDRESS_LINES);
+        if (splitByColon.length > 1 && splitByColon.length <= MAX_ADDRESS_LINES + 1) {
```

Comment:
I know it's not on this line, but GitHub won't let me comment on the actual line: could we change the example in the JavaDoc comment on Line 11 to remove the last semi-colon on

Suggestion below:
```
 * E.g. {@code ADR++US:FLAT1:12 BROWNBERRIE AVENUE::LEEDS++LS18 5PN'} has no address parts 3 or 5.
 ```

Reply:
Yep, sure